docs/src/validation_services.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Other than the last paragraph, I'm not sure how useful this doc is.  The hardware specs are rapidly changing and folks have found setups that work well for <$200/month in certain data centers.  Proposing a crazy expensive rig is a bit out of character for this otherwise academic-only section.

Reply:
Agreed, I was on the fence about this section too. I think we can leave it out. Maybe move the table to an FAQ (or make a new table).